src/content/guidelines/color/overview.md

```diff
@@ -0,0 +1,185 @@
+
+### Hover
+Hover states have their own tokens. In the IBM default themes, these colors fall outside of the core palette steps and act as "half steps" between two adjacent colors. The White theme shares the same hover value as the Gray 10 theme; the two dark theme backgrounds share a hover value as well.
+
+### Active
+The active state is two full steps lighter for values between 100 and 70 and two full steps darker for values between 60 and 10. For example, the Blue 60 active state is Blue 80. The exception is that White shares the same active state as Gray 10, and Black shares the same active state as Gray 100. 
+
+### Selected
+The selected state is one full step lighter for values between 100 and 70, and one full step darker for values between 60 and 10. For example, the Blue 60 selected state is Blue 70. The exception is that White shares the same selected state as Gray 10, and Black shares the same selected state as Gray 100. 
+
+### Focus
+The focus state usually appears as a Blue 60 border in the light theme and as a White border in the dark theme.
+
+### Disabled
+Disabled components are unavailable for interaction, so they don't receive hover or focus and are not subject to WC3 contrast compliance standards. For the dark themes, disabled values are two full steps lighter than their respective background. For light themes, they are two full steps darker. The exception is that the White theme disabled value is Gray 30.     
+
+![Interactive states light theme](images/Interactive_states_light_new.png)
+
+_Light theme interactive tokens_
+
+
+![Interactive states dark theme](images/Interactive_states_dark_new.png)
+
+_Dark theme interactive tokens_
+
+
+## Digital accessibility
+
+Using various forms of contrast is the most important consideration when making user-friendly color and interface choices. Awareness of standards and best practices is the key to accessible color selection.
+
+The color families in the IBM palette contain ten values from 10 to 100. White and Black sit outside those values. Black text is WCAG AA accessible on colors ranging from 10 to 50. White text is accessible on colors from 60 to 100.
+
+![Digital accessibility](images/accessibility.png)
+
+Beyond black and white, the IBM color palette provides a range of accessible combinations. Subtracting the foreground value from the background value (or vice versa) helps determine whether that color combination meets the WCAG AA contrast ratio success criteria. If the difference between two values is 50 or greater, the colors are accessible. Anything below a difference of 50 may fail accessibility standards.
+
+![Digital accessibility](images/accessibility_02.png)
+
+_Blue color family, 60 background_
+
+For a complete look at color-related accessibility topics in Carbon, visit the [Accessibility](/guidelines/accessibility/color) section.
```

Comment:
wondering if we want to just keep color blindness stuff in two places... like some of the theme content